packages/server/src/fhir/operations/export.ts

```diff
@@ -113,12 +113,14 @@ function getResourceTypesByExportLevel(exportLevel: string): ResourceType[] {
   return getResourceTypes();
 }
 
+const unexportedResourceTypes = ['CodeSystem', 'SearchParameter', 'StructureDefinition', 'ValueSet'];
+
 function canBeExported(resourceType: string): boolean {
   if (resourceType === 'BulkDataExport') {
```

Comment:
Now that we're introducing `unexportedResourceTypes`, should we move the `BulkDataExport` into that?

Then this method becomes `return !protectedResourcesTypes.includes(resourceType) && !unexportedResourceTypes.includes(resourceType)`